### PR TITLE
Update deprecated code from flutter 3.19

### PR DESCRIPTION
**Pull request by ellet0**
## Description

*Update the project the project to work with flutter 3.19*

The CI always use the latest version, the latest stable version of Flutter 3.19 deprecate some of the code that we are using in the project so I had to update it, I'm not sure if the new api is exist or compatible with flutter 3.16

if not then we need to update the minimum version of flutter from 3.16 to 3.19

## Checklist

- [ ] I read the [Contributor Guide](../CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [ ] I titled the PR using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0).
- [x] I did not modify the `CHANGELOG.md` nor the plugin version in `pubspec.yaml` files.
- [x] All existing and new tests are passing.
- [x] I have run the commands in `./scripts/before_push.sh` and it all passed successfully

## Breaking Change

Does your PR require plugin users to manually update their apps to accommodate your change?

- [ ] Yes, this is a breaking change (please indicate that with a `!` in the title as explained in [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0)).
- [x] No, this is *not* a breaking change.

### CODE REVIEW

**Review by singerdmx — APPROVED** [review]

File: dart_quill_delta/lib/src/delta/delta.dart [hunk 1]
```diff
@@ -147,7 +147,7 @@ class Delta {
   Operation get last => operations.last;
 
   @override
-  bool operator ==(dynamic other) {
```
Comment (ellet0) [inline dart_quill_delta/lib/src/delta/delta.dart:150]: I don't think we should use dynamic in the whole project, just to be more safe

Comment (singerdmx) [reply dart_quill_delta/lib/src/delta/delta.dart:150]: yeah

**Comment by ellet0** [comment]
I think there is a bug with the new updated code

image

I'm not sure though

**Comment by ellet0** [comment]
This [commit](https://github.com/singerdmx/flutter-quill/commit/f7dcd55931c166e163392cc7ade91e414bcdecea) should fix some changes of this PR

**Pull request merged**